nixos/modules/misc/ids.nix

```diff
@@ -356,6 +356,7 @@ in
       rstudio-server = 324;
       localtimed = 325;
       automatic-timezoned = 326;
+      readarr = 327;
```

Comment:
Please do not add new uuid unless absolutely necessary.

Did you follow a guide or wiki article we need to update?

Reply:
I just looked at what was done for other *arr apps such as lidarr,sonarr,radarr etc. as they had their uids/gids here I did the same for the smoke-tests to work.

Reply:
@SuperSandro2000 is there a preferred approach? Happy to help migrate the other `*arr` apps if I know what to do.

Reply:
Just define a user and group under users.

Tests shouldn't require fixed uids and gids.

Reply:
Done I have removed the fixed uids/gids :)